fuzz/fuzz_http3serverreq.cc

```diff
@@ -103,7 +111,23 @@ static int end_stream(nghttp3_conn *conn, int64_t stream_id,
                       void *conn_user_data, void *stream_user_data) {
   auto fuzzed_data_provider = static_cast<FuzzedDataProvider *>(conn_user_data);
 
-  return fuzzed_data_provider->ConsumeBool() ? NGHTTP3_ERR_CALLBACK_FAILURE : 0;
+  if (fuzzed_data_provider->ConsumeBool()) {
+    return NGHTTP3_ERR_CALLBACK_FAILURE;
+  }
+
+  if (fuzzed_data_provider->ConsumeBool()) {
+    return 0;
+  }
+
+  auto name = fuzzed_data_provider->ConsumeRandomLengthString();
+  auto value = fuzzed_data_provider->ConsumeRandomLengthString();
+
+  const nghttp3_nv nva[] = {
+    MAKE_NV(name.c_str(), value.c_str()),
```

Comment:
name.c_str() is a pointer to char, and sizeof(...) returns the size of pointer, which is not what you want here.
MAKE_NV is intended for string literals.

Reply:
Good catch. Fixed.